dist/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please implement the requirement asked by the previous reviewer, which asked to follow the wireframe for the leaderboard, refer to his pictures and link for more info.